commands/command_edit.cpp

```diff
@@ -1285,6 +1286,56 @@ void EditCylinder::undo()
 	redo();
 }
 
+EditSensors::EditSensors(int cylIndex, divecomputer *dcIn, std::vector<int> sensors)
+	: EditDivesBase(true), dc(dcIn)
+{
+	if (dives.empty())
+		return;
+
+	if (dives.size() == 1)
+		setText(Command::Base::tr("Edit sensors"));
+	else
+		throw std::runtime_error("Can only edit sensors for current dive computer");
```

Comment:
In general we don't throw runtime errors. Instead we try to fail gracefully (for some value of that term).
I'm not sure how you would get here with more than one dive selected... I'd prefer to disable THAT instead of trying to bail from here.

Reply:
Yes, unfortunately Qt is fundamentally not exception save. And owing to its completely broken ownership model it can never be made exception save.

I will try to look at the rest of the PR later this week.

Reply:
Yea. I basically just added it as an assert so I would notice if I triggered it. The command is currently being hard-coded to just be for one dive. It feels a bit strange when you have multiple dives selected and change the sensor and it just changes the current dive computer. It doesn't trigger the warning about multiple dives being edited and maybe there is something else with similar behavior. I've been looking into hiding or making the sensors field uneditable when multiple dives are selected but haven't figured that out yet.